README.md

```diff
@@ -139,6 +140,24 @@ Enable syntax highlighting by adding `files.associations` in `settings.json`, to
 ```
 There's notification message at right side of status bar.
 
+## Samples for azure_rm_resource (preview)
+
+This option provides easy an access to the samples available from Azure REST API Specifications.
```

Comment:
This option provides sample code snippets for [azure_rm_resource](link here) module, those samples are from Azure Rest API  spec.